### PR TITLE
Expand testing to include most of CMake's cuda tests

### PR DESCRIPTION
**Pull request by robertmaynard**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -61,14 +61,18 @@ outputs:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
         - cmake
+        - cuda-driver-dev
+        - git # [linux]
```
Comment (bdice) [inline recipe/meta.yaml]: FYI, the linter will probably complain once this is merged: you need two spaces before the `# [linux]`.
```suggestion
        - git  # [linux]
```

**Review by bdice — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -61,14 +61,18 @@ outputs:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
         - cmake
+        - cuda-driver-dev
+        - git  # [linux]
         - ninja
       files:
         - test.cu
         - CMakeLists.txt
+        - run_cmake_cuda_tests.sh # [linux]
```
Comment (adibbley) [inline recipe/meta.yaml]:
```suggestion
        - run_cmake_cuda_tests.sh  # [linux]
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -61,14 +61,18 @@ outputs:
...
       commands:
         - nvcc --version
         - nvcc --verbose test.cu
         - cmake -S . -B ./build -G=Ninja && cmake --build ./build -v
+        - bash ./run_cmake_cuda_tests.sh # [linux]
```
Comment (adibbley) [inline recipe/meta.yaml]:
```suggestion
        - bash ./run_cmake_cuda_tests.sh  # [linux]
```

**Review by adibbley — COMMENTED** [review]
Looks like these also need another space for the linter

**Pull request merged**